components/diagnose/options.go

```diff
@@ -16,6 +15,8 @@ type Op struct {
 	diskcheck bool
 
 	dmesgCheck bool
+
+	checkIb bool
```

Comment:
```suggestion
	checkInfiniBand bool
```
Keep the same style as command.go.

Reply:
done